### PR TITLE
Cheatsheets description fix

### LINKED ISSUE #1776: Description missing in IRC cheat sheet
Description is missing in [IRC cheat sheet](https://duckduckgo.com/?q=IRC+cheat+sheet&ia=cheatsheet).
@moollaza Shall I add it ?

### PR DESCRIPTION
**Pull request by gautamkrishnar**
fixes #1776

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
cc: @moollaza

File: share/goodie/cheat_sheets/json/irc.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "id": "irc_cheat_sheet",
     "name": "IRC Cheat Sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/irc.json]: Let's remove "Cheat Sheet" from the `name` of these

**Comment by moollaza** [comment]
Looks like the branch has conflicts. Please resolve them so this is mergable

**Comment by gautamkrishnar** [comment]
Okay

**Comment by gautamkrishnar** [comment]
@moollaza can it now be merged?

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**